### PR TITLE
Adding error codes to image package

### PR DESCRIPTION
**Pull request by brahmaroutu**
Signed-off-by: Srini Brahmaroutu srbrahma@us.ibm.com

### CODE REVIEW

File: errors/image.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package errors
+
+// This file contains all of the errors that can be generated from the
+// docker/daemon component.
```
Comment (runcom) [inline errors/image.go]: is this vallid here since we're under `image/`?

**Comment by runcom** [comment]
just a nit, LGTM otherwise

**Comment by duglin** [comment]
LGTM once runcom's nit is fixed

File: errors/image.go (hunk 1 above)
Comment (brahmaroutu) [inline errors/image.go]: Cut&paste, Fixed.

**Comment by tiborvass** [comment]
LGTM

**Pull request merged**